README.md

```diff
@@ -265,6 +266,19 @@ WARN[0000] CPU limit exceeded, it is set to 1 but it must not exceed 500m. Pleas
 WARN[0000] Memory limit exceeded, it is set to 512Mi but it must not exceed 125Mi. Please adjust it!
 ```
 
+<a name="all" />
+
```

Comment:
This is a great feature and I think will be the audit that 99% of all the people will run 99% of the time. Thus, it needs to be above all other audits, right next to `autofix`